app/helpers/account_helper.rb

```diff
@@ -0,0 +1,50 @@
+module AccountHelper
+
+  ## Helpers available to accounts. 
+  ## FIXME - Move to 
+  ### Setup item status
+  ITEM_STATUS = YAML.load_file("#{Rails.root}/config/voyager_item_status.yml") rescue {}
+
+  def item_status_to_label item
+    ITEM_STATUS[item['statusCode'].to_i]
+  end
+
+  def format_date raw_date
+    raw_date.to_time.in_time_zone.strftime('%B %e %Y at %l:%M %p')
+  end
+
+  def format_block_statement block_message
+    if block_message == "odue_recall_limit_patron"
+      I18n.t('blacklight.account.overdue_block')
+    end
+  end
+
+  def format_hold_cancel item
+    "item-#{item["itemID"]}:holdrecall-#{item["holdRecallID"]}:type-#{item["holdType"]}"
+  end
+
+  def format_renew_string item
+    "#{item['itemId']}"
+  end
+
+  def display_account_balance fine_fee
+    "#{fine_fee['balanceTotal']}".strip.gsub(/\n\s+/, ' ')
+  end
+
+  def message_status item
+    unless item[:messages].nil?
```

Comment:
unless, else is bad style - do `if !item[:messages].nil?` instead if there's going to be an else clause